### ISSUE TITLE
NODES : Orphan Nodes are NOT reflected in FGA 

### ISSUE THREAD

**Issue by BecOzIcan** [Portree-Kid/flightgear-airports#8]
UUEE file contains a series of orphan (not connected to anything) node in the north east corner of the apron. 

Orphan nodes not plotted on FGA 

Use Case : 
Import eAIP data (ParkPos Lat Lon) for set a first wireframe to build upon.

**Comment by Portree-Kid** [comment]
Currently FGA doesn't have nodes except parking. All Taxinodes belong to a Segment. Shouldn't these Orphans be added as a ParkPos?

**Comment by BecOzIcan** [comment]
I guess you are right conceptually; these are parkpos in the AIP and I should have ingested them as such, not asa nodes. 

In your data model, are nodes 'children' of segments? Would the same apply to ParkPos? ie are segments the only main/parents elements with either a node or Parkpos child at each end? 

Practically, if I was to edit UUEE in FGA, upon saving, would the orphan nodes be removed from the xml

**Comment by Portree-Kid** [comment]
Yes orphans are generally removed. EGPO had some random ones all over the place. If they are not connected I don't see the point in keeping them. Maybe add a question on import? "Unconnected nodes found. Convert to parking?"

**Comment by BecOzIcan** [comment]
Happy with that.
1. Convert on Import (I wouldn't even offer any option, just proceed)  ie "Will be converted to ParkPos" 
2. Removed on Export
